admin_ui/src/App.vue

```diff
@@ -50,17 +54,11 @@ export default Vue.extend({
             function (error) {
                 if (error.response && error.response.status == 401) {
                     console.log("Login required")
-                    let nextURL = app.$route.path
-                    if (nextURL !== "/login") {
-                        setTimeout(function () {
-                            app.$router.push({
-                                name: "login",
-                                query: {
-                                    nextURL: nextURL,
-                                },
-                            })
-                        }, 0)
-                    }
+                    setTimeout(function () {
+                        app.$router.push({
+                            name: "login",
+                        })
+                    }, 0)
```

Comment:
What was the issue here? Are we unable to access `app.$route.path`?

Reply:
There is no issue but new way does the same thing with much cleaner url (`http://piccolo_admin/#/login/` vs `http://piccolo_admin/#/login/?nextURL=/login/`)

Reply:
The only advantage with the old approach, is let's say you're on a page and you go away and make a cup of tea. You come back, and your session has expired. When you login, it will take you back to the page you were previously on.

Reply:
You are right. I moved back to the old approach.